cocotb/share/lib/fli/FliCbHdl.cpp

```diff
@@ -140,6 +144,53 @@ int FliStartupCbHdl::arm_callback()
     return 0;
 }
 
+
+static void get_args(gpi_sim_info_t *info)
```

Comment:
Please add documentation to this function. It's a sufficiently obscure way to pass plusargs to a simulator to warrant a couple lines of docs.